src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -35,18 +38,57 @@ namespace Windows.UI.Xaml.Controls
 {
 	public partial class ScrollViewer : ContentControl, IFrameworkTemplatePoolAware
 	{
-		internal const string ScrollContentPresenterPartName = "ScrollContentPresenter";
+		private static class Parts
+		{
+			public static class Uwp
+			{
+				public const string ScrollContentPresenter = "ScrollContentPresenter";
+				public const string VerticalScrollBar = "VerticalScrollBar";
+				public const string HorizontalScrollBar = "HorizontalScrollBar";
+			}
+
+			public static class WinUI3
+			{
+				public const string Scroller = "PART_Scroller";
+				public const string VerticalScrollBar = "PART_VerticalScrollBar";
+				public const string HorizontalScrollBar = "PART_HorizontalScrollBar";
+			}
+		}
+
+		private static class VisualStates
+		{
+			public static class ScrollingIndicator
+			{
+				public const string None = "NoIndicator";
+				public const string Touch = "TouchIndicator";
+				public const string Mouse = "MouseIndicator";
+				// public const string MouseFull = "MouseIndicatorFull"; // No supported yet
+			}
+
+			public static class ScrollBarsSeparator
```

Comment:
Will this fail with fluent styles ?

Reply:
If the "fluent style" is compatible with the current `ScrollViewer`, it will remain compatible with the updated style.

Reply:
IIRC there's no fluent style for the `ScrollViewer` itself, there seems to be one for `ScrollBar` however https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Themes/WinUI/Resources/PriorityDefault/ScrollBar_themeresources.xaml